site/content/recipes/find_a_reviewer.md

```diff
@@ -0,0 +1,160 @@
+I ask about here?"](https://codereview.stackexchange.com/help/on-topic) page to
+determine whether it could suit your needs.
+
+## Step 2: Criteria for Screening {#criteria}
+
+Once you have identified a potential reviewer and before you contact them, you
+should consider whether they will satisfy some minimum thresholds to be a
+suitable reviewer. These criteria equally apply when deciding whether to
+advertise to a larger group.
+
+1. Do they have some level of experience with the programming language(s) of
+   your code? They do not need to be experts in the language(s), but they
+   should not be absolute beginners. Code review is an opportunity to learn
+   _more_ about a programming language, not to learn the language itself. 
+2. Do they have some degree of knowledge about your research domain? This is
+   not strictly necessary, but can be desirable. Domain knowledge will help
+   your reviewer more quickly grasp the context of what your code is doing and
+   therefore address the code itself more directly. Researchers or developers
+   outside your domain can certainly offer valuable insight, but it might just
+   take a bit longer. We are all for cross-disciplinary interactions.
+3. Are they familiar with the libraries or tools that you use? As for the
+   previous criterion, this one isn't strictly necessary, but it can have quite
+   a large impact upon the value of the review. The importance of this
+   criterion scales with the extent to which your code relies on another
+   library or tool and how essential it is to the functionality of your code.
+
+## Step 3: Contact candidate or advertise {#contact}
+
+Everyone has their own way of writing messages, and we don't want to be too
+prescriptive about how you reach out to a potential reviewer. If you already
+know the reviewer, this step should be straightforward, but if not, then it can
+be a little stressful going out on a limb to contact someone you don't know or
+advertise to a group of people, especially if they are more senior than you.
+Our advice: don't overthink it, and stay concise and to the point. A casual and
+less formal atmosphere for the code review is desirable, so if you find
+yourself writing a stilted and overly-formal message to a candidate, then it
+might be worth thinking whether it will be a good fit.
+
+Consider including the following information in any message or advertisement
+you compose:
+
+- Context about _why_ you are looking to have your code reviewed. The
+    motivations section (**TODO** add link) on this website might be helpful to
+    identify the reasons that motivated you to get your code reviewed.
+- A brief summary of what your code does and any particular aspects you want
+    addressed.
+- [Any of the criteria for a suitable reviewer](#criteria) that you weren't
+    able to answer yourself before reaching out.
+- An idea of the time commitment (**TODO** what is our estimate for how long the
+    whole process will take?)
```

Comment:
Didn't the above section coming, definitely a good idea to provide guidelines for people to contact reviewers.
Time commitment is super important but hard to estimate! That's something we might want to cover in the resources section of the website

Reply:
I agree, estimating time for a review will be tricky, but we need to provide some sort of bounding values to use as estimates. I would hope we can make a statement like: "If you follow the steps outlined on this website (especially related to selecting an appropriate segment of code), then the code review should take between [lower bound estimate] and [upper bound estimate]." We need to have a discussion to get those numbers, perhaps from experience.

Reply:
Consider making the examples below as "vignettes" and having a visual offset for each so the reader can see that each is self-contained (and can be skipped if that's appropriate).

Reply:
I have added the examples as collapsable boxes. It looks good!